### ISSUE TITLE
TypeError: Unknown color 'orange'

### ISSUE THREAD

**Issue by max-te** [SamSchott/maestral#136]
When an update is available maestral crashes in
https://github.com/SamSchott/maestral/blob/aeb54b6cedbd9ba85a50cb6e1fb2649a3d4c4d94/maestral/cli.py#L147-L150
with this output:
```
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/click/termui.py", line 498, in style
    bits.append("\033[{}m".format(_ansi_colors[fg]))
KeyError: 'orange'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/maestral", line 11, in <module>
    load_entry_point('maestral==1.0.0.dev5', 'console_scripts', 'maestral')()
  File "/usr/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 1256, in invoke
    Command.invoke(self, ctx)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/maestral/cli.py", line 388, in main
    check_for_updates()
  File "/usr/lib/python3.8/site-packages/maestral/cli.py", line 147, in check_for_updates
    click.secho(
  File "/usr/lib/python3.8/site-packages/click/termui.py", line 547, in secho
    message = style(message, **styles)
  File "/usr/lib/python3.8/site-packages/click/termui.py", line 500, in style
    raise TypeError("Unknown color '{}'".format(fg))
TypeError: Unknown color 'orange'
```

According to click these colors are supported:
https://github.com/python-trio/trio-click/blob/bac4f5404749772f315b90ea38fe02de81681835/click/termui.py#L391-L410

**Comment by SamSchott** [comment]
Thanks for catching this! No idea how this could happen...

**Comment by SamSchott** [comment]
Ok, this is fixed in v1.0.2. It is sadly ironic that those on v1.0.0 will only be affected by the crash when an update is available that fixes it 😕

I'll leave the issue open for know, for those you search for it...
